modules/fedimint-ln/src/lib.rs

```diff
@@ -979,7 +985,11 @@ async fn block_height(interconnect: &dyn ModuleInterconect) -> u32 {
     // This is a future because we are normally reading from a network socket. But for internal
     // calls the data is available instantly in one go, so we can just block on it.
     let body = interconnect
-        .call("wallet", "/block_height".to_owned(), Default::default())
+        .call(
+            LEGACY_HARDCODED_INSTANCE_ID_WALLET,
```

Comment:
We need to figure out how to make this non-hard coded. Config field referencing the other module?

Reply:
This whole `IFederationApi` has to go IMO: https://github.com/fedimint/fedimint/issues/1169